### PR TITLE
Bug 1337896 - Add FENNEC_ACTIVITY_STREAM_TOPSITES_LOADER_TIME_MS and FENNEC_TOPSITES_LOADER_TIME_MS to android_clients

### PR DESCRIPTION
**Pull request by fbertsch**
@maurodoglio, now in an actually readable format :)

Question is: Do we remove this from the mozilla-services/data-pipeline repo, and have the airflow job read from here?

### CODE REVIEW

**Comment by maurodoglio** [comment]
Yes, the airflow job should point directly to this repo

**Comment by fbertsch** [comment]
Whoops, forgot to mention - I tested this on 100% of data from 2017-02-01, and everything worked well. The new columns are just arrays with the bucket counts.

File: etl/android-clients.kp/knowledge.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+```
+Create a set of pings from "saved-session" to build a set of core client data. Output the data to CSV or Parquet.
+
+This script is designed to loop over a range of days and output a single day for the given channels. Use explicit date ranges for backfilling, or now() - '1day' for automated runs.
+
+
+```python
+channels = ["nightly", "aurora", "beta", "release"]
+
+batch_date = '20170201'#os.environ.get('date')
```
Comment (maurodoglio) [inline etl/android-clients.kp/knowledge.md]: you probably forgot to revert this change

**Comment by fbertsch** [comment]
Good catch, fixed.
